plugins/wiki/main.go

```diff
@@ -76,11 +76,23 @@ func Handle(source mautrix.EventSource, evt *event.Event) {
 					j.Extract = "We were unable to define this term."
 				}
 				message := fmt.Sprintf(msgformat, j.Title, j.Extract)
-				matrix.Client.SendMessageEvent(evt.RoomID, event.EventMessage, format.RenderMarkdown(message, false, true))
```

Comment:
WAIT!
do `!wiki en Litecoin Bitcoin` on my bot and on your fork.
It will edit the same message twice, it should check if it's already edited and send new message in that case

Reply:
That is not going to happened, unless someone is running selfbot from two different machines/containers
`Another process is using this Badger database.: resource temporarily unavailable`
I'm getting this error when trying to run two instances at the same time

Reply:
I mean that in current (mrcyjanek/master) configuration it will send two messages, and on your fork it will edit one message twice